honeybee_radiance/cli/dc.py

```diff
@@ -287,3 +287,95 @@ def rfluxmtx_command_with_postprocess(
         sys.exit(1)
     else:
         sys.exit(0)
+
+
+@dc.command('coeff')
+@click.argument(
+    'octree', type=click.Path(exists=True, file_okay=True)
+)
+@click.argument(
+    'sensor-grid', type=click.Path(exists=True, file_okay=True)
+)
+@click.argument(
+    'sky-dome', type=click.Path(exists=True, file_okay=True)
+)
+@click.option(
+    '--sensor-count', type=click.INT, show_default=True,
+    help='Number of sensors in sensor grid file. Number of sensors will be parsed form'
+    ' the sensor file if not provided.'
+)
+@click.option(
+    '--rad-params', show_default=True, help='Radiance parameters.'
+)
+@click.option(
+    '--rad-params-locked', show_default=True, help='Protected Radiance parameters. '
+    'These values will overwrite user input rad parameters.'
+)
+@click.option(
+    '--output', '-o', show_default=True, help='Path to output file. If a relative path'
+    ' is provided it should be relative to project folder.'
+)
+@click.option(
+    '--output-format', help='Output type for converted results. Valid inputs are a, f '
+    'and d for ASCII, float or double.', type=click.Choice(['a', 'f', 'd']), default='f',
+    show_default=True, show_choices=True
+)
+@click.option(
+    '--keep-header/--remove-header', ' /-h-', is_flag=True, default=True,
+    help='A flag to keep or remove the header from the output file.'
+)
+@click.option(
+    '--dry-run', is_flag=True, default=False, show_default=True,
+    help='A flag to show the command without running it.'
+)
+def rfluxmtx_command_without_postprocess(
```

Comment:
Is it possible to add an optional flag to our existing rfluxmtx command, assuming we have one, instead of creating a new command? `--postprocess/--no-postprocess`?

Reply:
I see the name may be a bit misleading. The existing command is with a sky matrix (argument) and optional postprocess. I don't need the sky matrix or any postprocessing.